maritalk/resources/api.py

```diff
@@ -1,5 +1,29 @@
 import requests
 from typing import List, Dict, Union
+from requests.exceptions import HTTPError
+from http import HTTPStatus
+
+
+class MaritalkHTTPError(HTTPError):
+    def __init__(self, request_obj):
+        self.request_obj = request_obj
+        try:
+            response_json = request_obj.json()
+            if "detail" in response_json:
```

Comment:
There will be an errror `api_message is undefined` when there is no key `detail` or `message`, right?